Source/Vehicles/Turrets/Turret/VehicleTurret.cs

```diff
@@ -1387,7 +1387,7 @@ public virtual void DrawAt(Vector3 drawPos, Rot8 rot)
 		{
 			if (!NoGraphic)
 			{
-				VehicleGraphics.DrawTurret(this, drawPos, Rot8.North);
```

Comment:
Can you link screenshots or a video? I don't think adding rotation textures for turrets will be this simple.